src/main.tsx

```diff
@@ -13,22 +13,25 @@ import ProductShop from "./pages/product-shop/product-shop.tsx";
 import { apiGetCategories, CategoriesModel } from "./api/api-get-categories.ts";
 
 export async function loader() {
-  const category : CategoriesModel[] | undefined = await apiGetCategories("https://lopi2-backend-5517f8f04d28.herokuapp.com/api/");
-  return category ;
-} 
+  const category: CategoriesModel[] | undefined = await apiGetCategories(
+    "https://lopi2-backend-5517f8f04d28.herokuapp.com/api/"
+  );
+  return category;
+}
 
 const router = createBrowserRouter([
   {
     path: "/",
     element: <MainPage />,
     errorElement: <ErrorPage />,
     loader: loader,
-    
+
     children: [
       {
         path: "category",
```

Comment:
plural form? as we are loading categories

Reply:
ok